exasol_sagemaker_extension/deploy.py

```diff
@@ -1,28 +1,55 @@
 import logging
 import click
-from exasol_sagemaker_extension.deployment.deploy_cli import main as scripts_deployer_main
-from exasol.python_extension_common.deployment.language_container_deployer_cli \
-    import language_container_deployer_main, slc_parameter_formatters, CustomizableParameters
-from exasol_sagemaker_extension.deployment.sme_language_container_deployer import SmeLanguageContainerDeployer
+from exasol.python_extension_common.cli.std_options import (
+    StdParams, StdTags, select_std_options, ParameterFormatters)
+from exasol.python_extension_common.cli.language_container_deployer_cli import (
+    LanguageContainerDeployerCli)
 
+from exasol_sagemaker_extension.deployment.sme_language_container_deployer import (
+    SmeLanguageContainerDeployer)
+from exasol_sagemaker_extension.deployment.deploy_create_statements import (
+    DeployCreateStatements)
 
-@click.group()
-def main():
-    pass
+CONTAINER_URL_ARG = 'container_url'
+CONTAINER_NAME_ARG = 'container_name'
 
+ver_formatter = ParameterFormatters()
+ver_formatter.set_formatter(CONTAINER_URL_ARG, SmeLanguageContainerDeployer.SLC_URL_FORMATTER)
+ver_formatter.set_formatter(CONTAINER_NAME_ARG, SmeLanguageContainerDeployer.SLC_NAME)
+formatters = {StdParams.version: ver_formatter}
 
-slc_parameter_formatters.set_formatter(CustomizableParameters.container_url,
-                                       SmeLanguageContainerDeployer.SLC_URL_FORMATTER)
-slc_parameter_formatters.set_formatter(CustomizableParameters.container_name,
-                                       SmeLanguageContainerDeployer.SLC_NAME)
+opts = select_std_options([StdTags.DB, StdTags.BFS, StdTags.SLC],
+                          exclude=StdParams.language_alias, formatters=formatters)
+opts.append(click.Option(['--to-print/--no-to-print'], type=bool, default=False))
+opts.append(click.Option(['--develop/--no-develop'], type=bool, default=False))
+opts.append(click.Option(['--deploy-slc/--no-deploy-slc'], type=bool, default=True))
+opts.append(click.Option(['--deploy-scripts/--no-deploy-scripts'], type=bool, default=True))
 
-main.add_command(scripts_deployer_main)
-main.add_command(language_container_deployer_main)
+
+def deploy(deploy_slc: bool, deploy_scripts: bool,  **kwargs):
+
+    if deploy_slc:
+        # Workaround for the issue#78 in PEC
+        if StdParams.path_in_bucket.name in kwargs and kwargs[StdParams.path_in_bucket.name] is None:
+            kwargs[StdParams.path_in_bucket.name] = ''
+
+        slc_deployer = LanguageContainerDeployerCli(
+            container_url_arg=CONTAINER_URL_ARG,
+            container_name_arg=CONTAINER_NAME_ARG)
+
+        extra_params = {StdParams.language_alias.name: 'PYTHON3_SME'}
```

Comment:
Ah, ok you hard code the alias for the moment. Until we fixed the scripts that should be good enough.

Reply:
Yes, exactly